core/mo-services-xml/src/main/resources/xml/ServiceDefPLATFORM.xml

```diff
@@ -1611,6 +1617,14 @@
                       name="AttitudeMode" service="AutonomousADCS" />
           </mal:extends>
         </mal:composite>
+	<mal:composite
```

Comment:
Do not mix the commits - inertial pointing is not part of selection of attitude determination mode.

Reply:
https://stackoverflow.com/questions/6217156/break-a-previous-commit-into-multiple-commits